ynput/OpenPype
# AfterEffects: set frame range and resolution

## Changelog Description
Frame information (frame start, duration, fps) and resolution (width and height) is applied to selected composition from Asset Management System (Ftrack or DB) automatically when published instance is created.

It is also possible explicitly propagate both values from DB to selected composition by newly added menu buttons.

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

!! Requires update of extension.

## Testing notes:
![aftereffects_extension](https://github.com/ynput/OpenPype/assets/4457962/60802ad2-7f5f-4803-9a99-b59261365265)
1. set different values in `Composition Settings` in resolution and frame start and duration
2. create publishable instance from this updated composition
3. compare modified values, should match what is in DB
4. Do this also explicitly by newly added buttons.
